Ryujinx.Ava/UI/Models/SaveModel.cs

```diff
@@ -41,11 +36,27 @@ public long Size
 
         public bool SizeAvailable { get; set; }
 
-        public string SizeString => $"{((float)_size * 0.000000954):0.###}MB";
+        public string SizeString => GetSizeString();
 
-        public SaveModel(SaveDataInfo info, HorizonClient horizonClient, VirtualFileSystem virtualFileSystem)
+        private string GetSizeString()
+        {
+            const int scale = 1024;
+            string[] orders = { "GB", "MB", "KB" };
```

Comment:
```suggestion
            string[] orders = { "GiB", "MiB", "KiB" };
```
In a way to follow what have been done in https://github.com/Ryujinx/Ryujinx/pull/3716